sdk/documenttranslation/azure-ai-documenttranslation/azure/ai/documenttranslation/_models.py

```diff
@@ -65,6 +65,12 @@ def _to_generated(self):
     def _to_generated_list(glossaries):
         return [glossary._to_generated() for glossary in glossaries]  # pylint: disable=protected-access
 
+    def __repr__(self):
+        return '''TranslationGlossary(glossary_url={}, file_format={},
```

Comment:
Is it possible to use "" strings here? The triple ''' will insert newlines into the string representation of the object.